### ISSUE TITLE
Unable to interact with any other IronBackpacks pack while one is open

### ISSUE THREAD

**Issue by jasondm** [gr8pefish/IronBackpacks#214]
Prevents nesting or un-nesting packs, or moving other packs around in the main inventory. No console/log updates when attempting to do so

1. Create two packs
2. Open one pack
3. Attempt to move second pack

- IronBackpacks: 3.0.4
- Minecraft: 1.12.2
- Forge: 14.23.1.2598
- Other Conflicting Mods (may be irrelevant to your issue): Inventory Tweaks 1.64

**Comment by TehNut** [comment]
Not really a bug. Just an incomplete feature. The nesting system hasn't been implemented yet, so there is a global blacklist of all backpack interactions, currently.

**Comment by gr8pefish** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Improve Blacklist](https://trello.com/c/HOX77ukQ/81-improve-blacklist)

**Comment by TehNut** [comment]
Not related to the blacklist. There is a hard check for `IBackpack` in `ContainerBackpack` with a TODO.

https://github.com/gr8pefish/IronBackpacks/blob/523f1bdee2d959e8315774e27f1ae3efe709d1be/src/main/java/gr8pefish/ironbackpacks/container/ContainerBackpack.java#L145-L146

**Comment by gr8pefish** [comment]
Right, this is for the nesting system. Okay, thanks for the clarification.

**Comment by gr8pefish** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Restore Nesting Upgrade](https://trello.com/c/CksQAThj/24-restore-nesting-upgrade)

**Comment by jasondm** [comment]
While nesting still isn't completely implemented, version 3.0.5-6 does fix the issue I mentioned so I'll close this issue.  Thanks for the fix and the mod, looking forward to new versions.

**Comment by TehNut** [comment]
We'll leave this open until Nesting is implemented.

**Comment by JTBlum** [comment]
I'm able to get one backpack inside another by shift-clicking the pack to nest with a backpack inventory open in 1.12, which doesn't technically have the nesting feature yet. While I like being able to do it, it seems like a bug given the instructions and state of the project.
